website/content/docs/agent-and-proxy/agent/apiproxy.mdx

```diff
@@ -6,14 +6,19 @@ description: >-
   for Vault's API.
 ---
 
-# Vault agent API proxy
+# Vault Agent API proxy
 
 Vault Agent's API Proxy functionality allows you to use Vault Agent's API as a proxy
 for Vault's API.
 
 ~> Note: This functionality will be deprecated in a future release. Please
 switch to using [Vault Proxy](/vault/docs/agent-and-proxy/proxy) for API proxying purposes, instead.
 
+~> Note: Unlike Vault Agent, [Vault Proxy](/vault/docs/agent-and-proxy/proxy) supports
```

Comment:
It doesn't support static secret caching through it's API proxy -- while templating it does output templates to files, which acts like a cache in case of downtime but, the cache subsystem of Agent doesn't support it, nor does the API proxy.

I'll try and reword this so it's clearer! Thanks!